src/components/Reactions/MiniQuickEmojiReactions.js

```diff
@@ -32,12 +32,13 @@ const propTypes = {
     onEmojiPickerClosed: PropTypes.func,
 
     ...withLocalizePropTypes,
-    preferredSkinTone: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+    preferredSkinTone: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
 
 };
 
 const defaultProps = {
     onEmojiPickerClosed: () => {},
+    preferredSkinTone: undefined,
```

Comment:
This won't work because preferredSkinTone is supposed to be a number or string right? 

https://github.com/Expensify/App/blob/db1026aec0688ddc3ae1fd666c9af2200139c81b/src/components/Reactions/getPreferredEmojiCode.js#L5

The default skin tone is -1 per [this](https://github.com/Expensify/App/blob/cec24816e63f6de82d195bace5c80883ff26ba00/assets/emojis.js#L56-L58). Maybe we should make that a CONST and have that be the default here?

Reply:
Yeah, that makes sense!